### PR TITLE
Added hover and click indications for merch-card

### PR DESCRIPTION
**Pull request by st-angelo-adobe**
Resolves: [MWPW-161440](https://jira.corp.adobe.com/browse/MWPW-161440)

Test URLs:
- Before: https://main--mas--adobecom.hlx.live/studio.html
- After: https://mwpw-161440--mas--adobecom.hlx.live/studio.html

### CODE REVIEW

File: studio/src/aem/render-view.js [hunk 1]
```diff
@@ -41,24 +41,55 @@ class RenderView extends LitElement {
...
-                ${selected
-                    ? html`<sp-icon-remove slot="icon"></sp-icon-remove>`
-                    : html`<sp-icon-add slot="icon"></sp-icon-add>`}
-            </div>
-        </merch-card>`;
+        let tooltipTimeout;
+        return html`<overlay-trigger placement="top"
+            ><merch-card
+                class="${selected ? 'selected' : ''}"
+                slot="trigger"
```
[the remark this answers is not in the document]
Comment (st-angelo-adobe) [reply studio/src/aem/render-view.js:61]: Well, in this case the card itself is the "something you click", and the tooltip is the "hover-content". So unless the card is slotted into "trigger" I don't even think it is rendered. Is there something wrong with this approach?

[the remark this answers is not in the document]
Comment (yesil) [reply studio/src/aem/render-view.js:61]: you are right :)

**Review by yesil — APPROVED** [review]

**Review by Axelcureno — APPROVED** [review]
Overall lgtm, although I would consider defining the event handlers for the tooltip as class methods and binding them in the constructor instead of having them inline within the render template for better performance. Right now it might not be a concern, but as we add more complexity to the rendering of the cards, it might become a problem.

**Comment by st-angelo-adobe** [comment]
> Overall lgtm, although I would consider defining the event handlers for the tooltip as class methods and binding them in the constructor instead of having them inline within the render template for better performance. Right now it might not be a concern, but as we add more complexity to the rendering of the cards, it might become a problem.
> 
> cc @3ch023 @npeltier @yesil

I made the handlers class methods. However not sure about binding them, it seems the `render-view` is correctly bound every time, and from [some reading](https://github.com/lit/lit-element/issues/226) it sounds like it's not necessarily needed in lit components? Let me know if there's something I'm missing.

Also, one thing to mention about performance is that maybe there should be a `render-view-item` component as well? For stuff like selection for instance. Right now whenever selection changes, `renderItem` is called for each card every time. Maybe the selection state (as well as other states local to a single card) could be managed in a separate component, without rerendering the whole list every time. I think this is also be a performance concern.

**Comment by afmicka** [comment]
@st-angelo-adobe could you please update the branch?

**Comment by st-angelo-adobe** [comment]
@afmicka Done.

**Pull request merged**